cmux-tui/bindings/go/raw/client_test.go

```
@@ -402,11 +402,138 @@ func TestSocketResolutionFallsBackWhenUnixPathIsTooLong(t *testing.T) {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() { _ = os.RemoveAll(bindDir) })
+	leafLength := len([]byte(path)) - len([]byte(bindDir)) - 1
+	if leafLength <= len(".sock") {
+		t.Fatalf("temporary bind path cannot reach the canonical byte length: %q", path)
+	}
+	bindPath := filepath.Join(bindDir, strings.Repeat("x", leafLength-len(".sock"))+".sock")
+	if len([]byte(bindPath)) != len([]byte(path)) {
+		t.Fatalf("temporary bind path length = %d, want %d", len([]byte(bindPath)), len([]byte(path)))
+	}
+	listener, err := net.Listen("unix", bindPath)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)